### PR TITLE
SimplicialSurfaceByDressGroup

### PR DESCRIPTION
**Pull request by aniemeyer**
This command allows the construction of a simplicial surface by its dress group

### CODE REVIEW

File: gap/PolygonalComplexes/constructors.gi [hunk 1]
```diff
@@ -460,7 +460,7 @@ __SIMPLICIAL_IntSetConstructor("VerticesInFaces", __SIMPLICIAL_AllTypes,
 
 #######################################
 ##
-## UmbreallaDescriptors 
+##  UmbreallaDescriptors
```
Comment (MeikeWeiss) [inline gap/PolygonalComplexes/constructors.gi:463]: Here is a typo.

File: gap/PolygonalComplexes/constructors.gi [hunk 2]
```diff
@@ -1636,3 +1636,148 @@ end);
...
+
+#######################################
+##
+##  DressGroups
+##
+
+
+#############################################################################
+##
+#F  SimplicalSurfaceByDressGroup . . . . . . . . .  .surface by dress group
```
Comment (MeikeWeiss) [inline gap/PolygonalComplexes/constructors.gi:1649]: The first F in this line is wrong.

**Review by MeikeWeiss — CHANGES_REQUESTED** [review]
Very nice! 
I have a question that I'm not sure about:  Isn't an (involution) check for t1*t2 missing after line 1715?

File: gap/PolygonalComplexes/constructors.gi (hunk 1 above)
Comment (aniemeyer) [reply gap/PolygonalComplexes/constructors.gi:463]: Thank you very much. t1*t2 moves around an edge and thus could contain cycles of length 2 or 4. This is checked in line 1736.

**Comment by aniemeyer** [comment]
Thank you very much. I have made the changes and have added to more examples for groups that are not dress groups of surfaces.

**Comment by MeikeWeiss** [comment]
Thank you! But I can't find the two new examples.

File: gap/PolygonalComplexes/constructors.gi (hunk 2 above)
Comment (fingolfin) [reply gap/PolygonalComplexes/constructors.gi:1649]:
```suggestion
#M  SimplicalSurfaceByDressGroup . . . . . . . . . . . surface by dress group
```

**Review by fingolfin — COMMENTED** [review]
I am not sure what's wrong here. But I noticed the following perhaps related to the issue: `DrawSurfaceToJavaScript` is sometimes invoked like this:
```
DrawSurfaceToJavaScript(oct, "doc/Octahedron_ThickEdges.html", printRecord);;
```
and sometimes like this:
```
DrawSurfaceToJavaScript(oct, "Octahedron_InnerCircle.html", printRecord);;
```
Note the missing `doc/` prefix in the filename. I am not sure if this is relevant or not; it seems to depend on the current directory

**Review by MeikeWeiss — APPROVED** [review]

**Review by TomGoertzen — APPROVED** [review]

**Pull request closed without merging**